markdown/dev/reference/api/stack/en.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Stack
+---
+
+A Stack object represents a layer within a pattern, holding one or
```

Comment:
See above about the stacks-as-layers thing

Reply:
Changed to:
A Stack object represents a collection of parts within a pattern. Stacks are used when laying out the pattern.